test/bigtest/tests/note-create-test.js

```diff
@@ -0,0 +1,171 @@
+          });
+        });
+      });
+
+      describe('and note title is missing', () => {
+        beforeEach(async () => {
+          await noteForm.noteTitleField.enterText('');
+        });
+
+        it('should display missing title error', () => {
+          expect(noteForm.hasTitleMissingError).to.be.true;
+        });
+
+        describe('and save button is clicked', () => {
+          beforeEach(async () => {
+            await noteForm.saveButton.click();
+          });
+
+          it('should not redirect to previous location', function () {
+            expect(this.location.pathname + this.location.search).to.equal('/eholdings/notes/new');
+          });
+        });
+      });
+
+      describe('and close button is clicked', () => {
+        beforeEach(async () => {
+          await noteForm.closeButton.click();
+        });
+
+        it('should display navigation modal', () => {
+          expect(noteForm.navigationModalIsOpened).to.be.true;
+        });
+
+        describe('and cancel navigation button was clicked', () => {
+          beforeEach(async () => {
+            await noteForm.clickCancelNavigationButton();
+          });
+
+          it('should close navigation modal', () => {
+            expect(noteForm.navigationModalIsOpened).to.be.false;
+          });
+
+          it('should keep the user on the same page', function () {
+            expect(this.location.pathname + this.location.search).to.equal('/eholdings/notes/new');
+          });
+        });
+
+        describe('and continue navigation button was clicked', () => {
+          beforeEach(async () => {
+            await noteForm.clickContinueNavigationButton();
+          });
+
+          it('should close navigation modal', () => {
+            expect(noteForm.navigationModalIsOpened).to.be.false;
+          });
+
+          it('should redirect to previous page', function () {
+            expect(this.location.pathname + this.location.search).to.equal(`/eholdings/packages/${providerPackage.id}`);
+          });
+        });
+      });
+
+      describe('and all required fields are entered correctly', () => {
+        beforeEach(async () => {
+          await noteForm.noteTitleField.enterText('some title');
+        });
+
+        describe('and save button is clicked', () => {
+          beforeEach(async () => {
+            await noteForm.saveButton.click();
+          });
+
+          it('should redirect to previous page', function () {
+            expect(this.location.pathname + this.location.search).to.equal(`/eholdings/packages/${providerPackage.id}`);
+          });
+        });
+      });
+    });
+
+    describe('when the note create page is visited using a link', () => {
+      beforeEach(function () {
+        this.visit('/eholdings/notes/new');
+      });
+
+      it('should redirect to eholdings root route', function () {
+        expect(this.location.pathname + this.location.search).to.equal('/eholdings?searchType=providers');
```

Comment:
I assume this is just the default location here (when coming directly from a link)?

Reply:
@carolegodfrey, yes, correct